tests/scripts/quiet/quiet

```diff
@@ -9,6 +9,17 @@
 # If you are debugging a build / CI issue, you can get complete unsilenced logs
 # by un-commenting the following line (or setting VERBOSE_LOGS in your environment):
 # VERBOSE_LOGS=1
+#
+# This script provides most of the functionality for the adjacent make and cmake
+# wrappers.
+#
+# It requires two variables to be set:
+#
+# TOOL       - the name of the tool that is being wrapped (with no path), e.g. "make"
+#
+# NO_SILENCE - a regex that describes the commandline arguments for which output will not
+#              be silenced, e.g. " --version | test ". In this example, "make test" will
```

Comment:
Minor: `make lib test` would be a better example, to illustrate that the regex doesn't need to match all the arguments.